guice-intro/src/main/java/com/baeldung/examples/guice/marker/Communicator.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.examples.guice.marker;
+
+/**
+ *
+ * @author Baeldung
+ */
+public interface Communicator {
+    

```

Comment:
Shouldn't this interface declare at least one method, in the event that you wanted to inject based on the interface rather than explicitly injecting the implementing class, e.g. DefaultCommunicator? That class has two public methods, so maybe this interface should declare them.